### PR TITLE
Mention package.json on the Jest part of the docs

### LINKED ISSUE #1674: Can the Readme for Enzyme/Jest be improved?
URL: [`https://github.com/airbnb/enzyme/blob/master/docs/guides/jest.md`](https://github.com/airbnb/enzyme/blob/master/docs/guides/jest.md)

screen shot 2018-06-07 at 13 20 56

I believe it could be useful for new comers to understand that the aforementioned snippet of configuration should be place inside of the project's `package.json` and that more information about alternatives can be found [in Jest's documentation](https://facebook.github.io/jest/docs/en/configuration.html).

### PR DESCRIPTION
**Pull request by jaysoifer**
Closes #1674

### CODE REVIEW

File: docs/guides/jest.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 ## Configure with Jest
 
-To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` to literally the string `<rootDir>` and the path to your setup file.
+To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` in the `package.json` file of the project to literally the string `<rootDir>` and the path to your setup file.
```
Comment (ljharb) [inline docs/guides/jest.md]: This isn't necessarily where the config lives - it might be in a separate config file.

Comment (jaysoifer) [reply docs/guides/jest.md]: Would it be acceptable to change it to:

```To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` in your config file (e.g.: the `package.json` file of the project) to literally the string `<rootDir>` and the path to your setup file.```

My interpretation is that making this explicit can help people.

Comment (ljharb) [reply docs/guides/jest.md]: that sounds good, but let’s also include an example that’s not package.json, so people will know that they have to find their config file themselves?

Comment (jaysoifer) [reply docs/guides/jest.md]: How about:

>To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` in your config file (check Jest's documentation for the [possibles locations of that config file](http://jestjs.io/docs/en/configuration)) to literally the string `<rootDir>` and the path to your setup file.

Comment (ljharb) [reply docs/guides/jest.md]: That sounds great!

**Comment by jaysoifer** [comment]
Okay, I believe this is it 😅 

Let me know if this is ok now

**Comment by ljharb** [comment]
Yes, that's part of the docs build process which is updated only on release.

**Comment by ljharb** [comment]
The following packages are now released:
 - [enzyme v3.4.0](https://www.npmjs.com/package/enzyme)
 - [enzyme-adapter-react-16 v1.2.0](https://www.npmjs.com/package/enzyme-adapter-react-16)
 - [enzyme-adapter-react-16.3 v1.0.0](https://www.npmjs.com/package/enzyme-adapter-react-16.3)
 - [enzyme-adapter-react-16.2 v1.0.0](https://www.npmjs.com/package/enzyme-adapter-react-16.2)
 - [enzyme-adapter-react-16.1 v1.0.0](https://www.npmjs.com/package/enzyme-adapter-react-16.1)
 - [enzyme-adapter-react-15 v1.0.6](https://www.npmjs.com/package/enzyme-adapter-react-15)
 - [enzyme-adapter-react-15.4 v1.0.6](https://www.npmjs.com/package/enzyme-adapter-react-15.4)
 - [enzyme-adapter-react-14 v1.0.6](https://www.npmjs.com/package/enzyme-adapter-react-14)
 - [enzyme-adapter-react-13 v1.0.4](https://www.npmjs.com/package/enzyme-adapter-react-13)
 - [enzyme-adapter-utils v1.5.0](https://www.npmjs.com/package/enzyme-adapter-utils)

**Pull request merged**